package.json

```diff
@@ -1240,7 +1220,9 @@
     "scripts": {
         "vscode:prepublish": "webpack --mode production",
         "lint": "eslint -c .eslintrc.js --ext .ts ./src",
-        "compile": "webpack --mode none",
+        "compile": "npm run compile:ext && npm run compile:view",
```

Comment:
Again, let's break these out into a separate "Curse All Webpacks, Jeeves" PR.

Reply:
Done, i also noted i didn't edit the `vscode:prepublish` and i guess we need to take care of that as well.